src/lib/apiUtils.js

```diff
@@ -283,6 +304,25 @@ export async function injectedCall(request, blockchain) {
         return _promptFail("injectedCall", request.id, request, reject);
     }
 
+    if (!isBlocked && visualizedParams) {
+        // account names will have 1.2.x in parenthesis now - check again
+        if (!blockedAccounts) {
+            try {
+                blockedAccounts = await blockchain.getBlockedAccounts();
```

Comment:
Why do we need to check here again after visualizing?

Reply:
The first attempt won't catch account names being used instead of account ids. 

We could remove the first check and rely on the visualize function to include `(account_id)` after the account name, since this is effectively the same check being run again if the first attempt didn't detect a blocked id.

Reply:
In which case do we have names in the payload? It's the json payload of a transaction.

Relying on the visualize function seems error prone in any case.

Reply:
Names in the payload are accepted by the bitshares-core and beet-js iirc, if we're just checking 1.2.x and the user uses their account name the check is bypassed. No?

True if the visualize function failed to communicate with the bitshares blockchain it could fail to include 1.2.x & thus also be bypassed.